cmd/globalAlignment/globalAlignment.go

```diff
@@ -32,6 +55,13 @@ func main() {
 		log.Fatalf("error, unable to read .fa files, check for > symbol before each name and that each fasta entry is only one line. Check to make sure there are only four bases or N, globalAlignment is unable to use anything outside A-T-G-C-N.\n")
 	}
 
+	//raven added this block
+	numSeqOne := CountSeqIdx(flag.Arg(0))
```

Comment:
This definitely works, but it may be able to be shortened a bit. There may be a good reason to use `fasta.NextFasta` but if `fasta.Read` would work instead then you could run a check like `if len(faOne) != 1 || len(faTwo) != 1 { Fatal }`. Just a suggestion, ready to approve as-is.

Reply:
Thank you! I think your suggestion will be more efficient, but can I ask you how the len(faOne) would work? fasta.Read returns []*Fasta; and when I googled len() I read that it counts the number of characters in a string. Is the idea to convert faOne (the Fasta output from fasta.NextFasta) into a string somehow (I couldn't do a simple conversion) and then use a variation of len() to count the number of lines, and if the number of lines in the Fasta != 2 then {Fatal}? I feel like I am missing something